pkgs/microcap/default.nix

```diff
@@ -41,12 +41,15 @@ in mkWindowsApp rec {
   issFile = ./setup.iss;
   
   winAppInstall = ''
-    cp "${issFile}" "$WINEPREFIX/drive_c/"
-    cp "${src}/setup.exe" "$WINEPREFIX/drive_c/"
-    $WINE "$WINEPREFIX/drive_c/setup.exe" /s /sms
+    mkdir                 "$WINEPREFIX/drive_c/microcap"
+    cp "${issFile}"       "$WINEPREFIX/drive_c/microcap/setup.iss"
+    cp -rf "${src}/."     "$WINEPREFIX/drive_c/microcap"
+    ls                    "$WINEPREFIX/drive_c/microcap"
+    $WINE                 "$WINEPREFIX/drive_c/microcap/setup.exe" /s /sms
```

Comment:
You probably got lucky here, because the wine process goes to the background immediately. You'll need to do the `wineserver -w` step right after the `$WINE ...` step. That way, the microcap directory won't get deleted until the setup.exe has finished; `wineserver -w` forces the script to wait for setup.exe to terminate.

Reply:
Although... it may be that the /sms flag is causing setup.exe to stay in the foreground; That's what that flag is for. That's what it does on Windows, so perhaps it works on Wine too.

Reply:
Thanks for the heads up, wasn't sure what ``wineserver -w`` did before you mentioned it.